cse/indexing/Dictionary.py

```diff
@@ -72,6 +75,9 @@ def __getitem__(self, key):
 
 
     def __setitem__(self, key, value):
+        """
+        value: pointer
```

Comment:
why not rename to `pointer`?

Reply:
Because it's the signature used in the python docs, so you recognize this method easier as a special one. 
The comment is a remnant of earlier tries, where value was a tuple of multiple values...

Reply:
ok, I can see your point.